packages/services/f-http/README.md

```diff
@@ -102,6 +105,39 @@ export default {
 }
 ```
 
+### Alternative Implementation
+If you would rather create the HTTPClient when you use it, thats fine too; its just means it can't be reused as easily and you will need to filter the configuration options down to the component.
```

Comment:
```suggestion
If you would rather create the HTTPClient when you use it, that's fine too; it's just means it can't be reused as easily and you will need to filter the configuration options down to the component.
```